PasscodeLock/PasscodeLockPresenter.swift

```diff
@@ -122,8 +122,9 @@ open class PasscodeLockPresenter {
             
         } else {
             
-            passcodeLockWindow.windowLevel = 0
+            passcodeLockWindow.windowLevel = UIWindowLevelNormal
             passcodeLockWindow.rootViewController = nil
+			passcodeLockWindow.isHidden = true
```

Comment:
extra tab?

Reply:
nope other indentations are with whitespaces
when I added this line my indentations are with tabs
should I refactor class to use tabs instead of whitespaces ?

Reply:
I don't think so :)